src/udiskslinuxfilesystem.c

```diff
@@ -1475,37 +1467,19 @@ handle_mount (UDisksFilesystem      *filesystem,
       goto out;
     }
 
-  escaped_fs_type_to_use       = udisks_daemon_util_escape_and_quote (fs_type_to_use);
-  escaped_mount_options_to_use = udisks_daemon_util_escape_and_quote (mount_options_to_use);
-  escaped_mount_point_to_use   = udisks_daemon_util_escape_and_quote (mount_point_to_use);
-  escaped_device               = udisks_daemon_util_escape_and_quote (udisks_block_get_device (block));
-
-  /* run mount(8) */
-  if (!udisks_daemon_launch_spawned_job_sync (daemon,
-                                              object,
-                                              "filesystem-mount", caller_uid,
-                                              NULL, /* GCancellable */
-                                              0,    /* uid_t run_as_uid */
-                                              0,    /* uid_t run_as_euid */
-                                              NULL, /* gint *out_status */
-                                              &error_message,
-                                              NULL,  /* input_string */
-                                              "mount -t %s -o %s %s %s",
-                                              escaped_fs_type_to_use,
-                                              escaped_mount_options_to_use,
-                                              escaped_device,
-                                              escaped_mount_point_to_use))
+  if (!bd_fs_mount (udisks_block_get_device (block), mount_point_to_use, fs_type_to_use, mount_options_to_use, NULL, &error))
```

Comment:
Please define a variable for the device name and store the result from ``udisks_block_get_device()`` there.